master/setup.py

```diff
@@ -464,7 +464,6 @@ def define_plugin_entries(groups):
     setup_args['extras_require'] = {
         'test': [
             'setuptools_trial',
-            'pep8',
```

Comment:
`pep8` was renamed `pycodestyle`, but `flake8` depends on it already, so we don't need to depend on it here.